src/components/BookList.js

```diff
@@ -17,22 +20,41 @@ const BookList = ({ books }) => {
   };
 
   return (
-    <ul>
+    <div className="book-list">
       {books.map((book) => (
-        <li key={book.itemId || uuidv4()}>
-          <h3>{book.title}</h3>
-          <p>
-            Author:
-            {book.author}
-          </p>
-          <p>
-            Category:
-            {book.category}
-          </p>
-          <button type="submit" onClick={() => handleRemoveBook(book.itemId)}>Remove Book</button>
-        </li>
+        <div key={book.itemId || uuidv4()} className="book-details">
+          <div className="book-info">
+            <p className="School-of Text-Style-9">
+              {book.category}
+            </p>
+            <h3 className="Title Text-Style-5">{book.title}</h3>
+            <div className="author-buttons">
+              <p className="Suzanne-Collins Text-Style-8">
+                {book.author}
+              </p>
```

Comment:
- [ ] Great job so far but I think here kindly fix the margin/padding between the book and the buttons and author since it is close to the given design, In your design, it has more space but we have to match the given design so please make it the same   💯✨⭐


- Your
![image](https://github.com/Eccb7/book-store/assets/58553711/9a2a27ca-a38d-4f8c-8281-5c8b0a82a90d)

- Given
![image](https://github.com/Eccb7/book-store/assets/58553711/87166033-ec34-4866-a1ec-49ed067de7ce)